src/api/app/controllers/webui/search_controller.rb

```diff
@@ -135,6 +149,32 @@ def set_parameters
     @owner_devel = '1' if params[:devel] == 'on'
   end
 
+  def search_issue
+    @search_issue = nil
+    @search_issue = params[:issue].strip if params[:issue].present?
+
+    @search_tracker = nil
+    @search_tracker = params[:issue_tracker] if params[:issue_tracker].present?
+  end
```

Comment:
Nitpick: I would move the search_tracker code out here and pass it directly in line 46:

```ruby
issue_tracker_name: params[:issue_tracker].presence
```

Reply:
Using `.presence` is definitely an improvement here.